code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -19,22 +19,25 @@
 	tier = XENO_TIER_ZERO
 	upgrade = XENO_UPGRADE_ZERO
 
-	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
```

Comment:
This just stops admins from being able to right click and interact with the hivemind.

Reply:
this fixed being able to rightclick hiveminds as a marine too, and PB

Reply:
Im still a little worried about this change.